.github/workflows/scorecards-analysis.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           persist-credentials: false
 
       - name: "Run analysis"
-        uses: ossf/scorecard-action@b614d455ee90608b5e36e3299cd50d457eb37d5f # Don't update this until they fix PR support
+        uses: ossf/scorecard-action@99c53751e09b9529366343771cc321ec74e9bd3d # Don't update this until they fix PR support
```

Comment:
This one should not be changed, as implied by the comment, or the scheduled build will break,

Reply:
Reverted back to original